virus/trojan_patchwork_virus.md

```diff
@@ -0,0 +1,107 @@
+* Viruses in the Virgaviridae family have alpha-like replication proteins that form a distinct phylogenetic "family" (Adams et al., 2009)
+* Viruses in the Virgaviridae family have a single-stranded RNA+ sense genome with a 3'-t-RNA-like structure witout a poly-A tail
+* Viruses in the Virgaviridae family have a central "canal"
+* Viruses in the Virgaviridae family have coat proteins of 19-23 kDa
+* Viruses in the Virgaviridae family are known to be very diverse and can infect a large host range of plant species (Adams et al., 2012)
+* Viruses in the Virgaviridae family are stable at higher temperatures (60-90 degrees C) (Adams et al., 2012)
+* The largest ORF in the Virgaviridae family is an alpha-like replication protien with methyltransferase and helicase domains (Adams et al., 2012)
+* the RdRp is expressed at the terminal part of this protien by readthrough of a leaky stop codon (Adams et al, 2012)
+* Replication in Viraviridae viruses is done in the cytoplasm (Adams et al., 2012)
+* Genuses within the Virgaviridae family include: furovirus, hordeivirus, pecluvirus, pomovirus, tabamovirus, and tobravirus (Adams et al., 2012)
+* Dozens of insect alpha-like viruses related to the virgaviridae-like viruses have been sequenced which suggest an evolutionary relationship between plant and insect viruses (Kondo et al., 2019; Shi et al., 2016)
+* Endogenous viral elements (EVEs) from RNA viruses have shown sequence similarity to ssRNA+viruses (such as virga-like virus) of plants that are integrated into many insect genomes suggesting that plant viruses may have integrated into insect genomes and vice versa (Cui & Holmes, 2012)
+* The viagravirus-related lineage consists of plant alpha-like viruses which includes the Virgaviridae family (Kondo et al., 2019)
+* Plant alpha-like viruses encode replication-associated proteins containing the same order of 5'cap methyltranferase, superfamily 1 helicase, and RdRp domains (Koonin et al., 1993)
+* Virgaviruses are mainly transmitted by soil-inhabiting organisms such as plasmodiophorid protists or nematodes (Andika et al., 2016)
+* There is evidence for virgavirus-like HEL domain sequence is located within long interspaced elements in lepidopteran species (moths) (Kondo et al., 2019)
+* It has been demonstrated that retrotransposons may be associated with RNA silencing activity in HEL domains as a reason for the high copy number of EVEs in lepidopteran (Kondo et al., 2019; Lazareva et al., 2015)
+* The vertical transmission efficiency of plant +ssRNA viruses (including virga-like viruses) is generally low except when transmitted through seeds or pollen (Andika et al., 2016)
+* Most transmission of plant +ssRNA viruses (including virga-like viruses) are transmitted horizontally (Witfield et al., 2015)
+* It is suggested that small RNAs derived from EVEs that have been integrated into the genome of insects may play a role in viral immunity in insects (Kondo et al., 2019)
+
```

Comment:
The Virus Genome figure is not included, either remove this section or include it with a figure legend please.